src/register/EmbeddableRegistrationPage.jsx

```diff
@@ -267,8 +327,19 @@ const EmbeddableRegistrationPage = (props) => {
         value,
         { name: formFields.name, username: formFields.username, form_field_key: name },
         !validationApiRateLimited,
+        false,
       );
     }
+    if (name === 'email') {
+      const payload = {
+        name: formFields.name,
+        email: formFields.email,
+        username: formFields.username,
+        password: formFields.password,
+        form_field_key: name,
+      };
```

Comment:
why do we need complete payload here?

Reply:
because this call will be made to backend for username suggestions i guess and i copied this payload from registration page.

Reply:
nope, the username suggestions call is made for the name field, not the email field. I think we don't need the payload.